mods/tuxemon/maps/spyder_paper_town.tmx

```diff
@@ -423,15 +423,39 @@
     <property name="act61" value="npc_face player,down"/>
     <property name="act70" value="translated_dialog spyder_papertown_firstfight"/>
     <property name="act80" value="start_battle spyder_billie"/>
-    <property name="act81" value="translated_dialog spyder_papertown_firstfight2"/>
+    <property name="act81" value="remove_npc spyder_billie"/>
     <property name="act90" value="set_variable firstfightdue:no"/>
     <property name="act91" value="set_variable teleport_faint:spyder_bedroom.tmx 7 6"/>
-    <property name="act94" value="pathfind spyder_billie,15,14"/>
-    <property name="act95" value="remove_npc spyder_billie"/>
-    <property name="act96" value="unlock_controls"/>
+    <property name="act92" value="set_variable firstfightend:yes"/>
     <property name="cond1" value="is variable_set firstfightdue:yes"/>
    </properties>
   </object>
+  <object id="425" name="First Fight - Win" type="event" x="384" y="144" width="16" height="16">
+   <properties>
+    <property name="act1" value="create_npc spyder_billie,26,12"/>
+    <property name="act10" value="npc_face spyder_billie,down"/>
+    <property name="act20" value="translated_dialog spyder_papertown_firstfight_win"/>
+    <property name="act30" value="pathfind spyder_billie,15,14"/>
+    <property name="act40" value="remove_npc spyder_billie"/>
+    <property name="act50" value="set_variable firstfightend:no"/>
```

Comment:
We use the `firstfightend` variable as a ratchet toggle - once the event is triggered, it won't trigger again.